dev/workflow/applications/gefs.py

```diff
@@ -97,6 +97,11 @@ def _get_app_configs(self, run):
         if options['do_aero_fcst']:
             configs += ['prep_emissions']
 
+        if options['nens'] > 0:
+            tasks += ['atmos_ensstat']
```

Comment:
I recommend adding `efcs` to here and remove the conditional on line 86-87 to avoid duplication. 

```suggestion
            configs += ['efcs', 'atmos_ensstat']
```